### DISCUSSION TITLE
Last Year's Controller Code

### DISCUSSION THREAD

**Discussion by pangoliff** [frc5567/2023-Robot#204] [Team Posts]
What issues, if any, was there with last year's controller class? Just asking so I know what to avoid.

**Comment by JooshO** [comment]
By controller, do you mean Pilot/Copilot controller or Gamepad?

**Comment by pangoliff** [comment]
Right now I'm looking at pilot/copilot controller

**Comment by JooshO** [comment]
I don't remember any functional issues but I'm sure there are improvements that can be made for structure/optimization

**Comment by pangoliff** [comment]
Alright, I'll keep that in mind.
